cpp/tests/centrality/eigenvector_centrality_test.cpp

```diff
@@ -277,8 +277,18 @@ INSTANTIATE_TEST_SUITE_P(
                       EigenvectorCentrality_Usecase{500, false, true},
                       EigenvectorCentrality_Usecase{500, true, false},
                       EigenvectorCentrality_Usecase{500, true, true}),
-    ::testing::Values(cugraph::test::File_Usecase("test/datasets/karate.mtx"),
-                      cugraph::test::File_Usecase("test/datasets/web-Google.mtx"),
+    ::testing::Values(cugraph::test::File_Usecase("test/datasets/karate.mtx"))));
+
+INSTANTIATE_TEST_SUITE_P(
+  file_large_pass,
```

Comment:
Why are we using "file_large_test" above and "file_large_pass" here? Better be consistent?

Reply:
Updated eigenvector centrality test names to match.